tools/test-proxy/startup-scripts/start-server.py

```diff
@@ -5,7 +5,8 @@
 import os
 
 CONTAINER_NAME = "ambitious_azsdk_test_proxy"
-IMAGE_SOURCE = "azsdkengsys.azurecr.io/engsys/ubuntu_testproxy_server:952205"
+# this image should be pinned when merged to main in your repo.
+IMAGE_SOURCE = "azsdkengsys.azurecr.io/engsys/testproxy:952205"
```

Comment:
Doe these not need to have the "-lin" or "-win" postfix? Where does that selection happen?

Reply:
Yeah. I just hit discovered an entirely new rabbit hole I need to go down to publish multiple platforms under a single tag. I'll update this to reflect appropriately.